src/index.ts

```diff
@@ -0,0 +1,52 @@
+import express = require('express')
+import influx from './config/database';
+const os = require('os')
+
+const app: express.Application = express()
+
+var bodyParser = require('body-parser');
+app.use(bodyParser.json()); // support json encoded bodies
+app.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
+
+
+app.get('/v1/query', function(req, res){
+  influx.query('select * from exodus_table')
```

Comment:
@HawkofSky are we pulling all from the table?

Reply:
We should query date as range e.g

`FROM "exodus_table" WHERE WHERE symbol = 'AAPL'
GROUP BY time(1d)`

Reply:
an example from here https://stackoverflow.com/a/39501566/2506749